modules/migration-mtc-release-notes-1-8-3.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/mtc-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="migration-mtc-release-notes-1-8-3_{context}"]
+= {mtc-full} 1.8.3 release notes
+
+[id=technical-changes-1-8-3_{context}]
+== Technical changes
+
+{mtc-full} ({mtc-short}) 1.8.3 has the following technical changes:
+
+.Support added for {oadp-short} 1.3
+
+{mtc-short} 1.8.3 adds support to {oadp-first} as a dependency of {mtc-short} 1.8.z. link:https://issues.redhat.com/browse/MIG-1520[(MIG-1520)]
+
+[id="resolved-issues-1-8-3_{context}"]
+== Resolved issues
+
+This release has the following major resolved issues:
+
+.Restic backup does not work properly when the source workload is not quiesced
+
+The HTML file is not migrated when migrating an application with a route. The Restic backup does not does not work properly when the source workload is un-quiesced.
```

Comment:
Read what you have written when you preview, not what you think you have written
```suggestion
The HTML file is not migrated when migrating an application with a route. The Restic backup does not function as expected when the source workload is unquiesced.
```